.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -14,7 +14,7 @@ assignees: ''
 Note to onduty:
 
 Use this issue to track a release from the initial canary release build through
-production. The community uses this issue to keep track of what is going on
+promotion to Stable. The community uses this issue to keep track of what is going on
```

Comment:
For clarity, change to
> through promotion to stable and LTS.

Reply:
Done.

Reply:
I don't see this change. Is this PR missing a commit?

Reply:
Odd; must have lost it during a rebase. Added anew